src/form/types/validators.ts

```diff
@@ -47,7 +47,11 @@ type CoreValidator =
   | 'duplicateIDNumber'
   | 'isValidDeathOccurrenceDate'
   | 'isMoVisitDateAfterBirthDateAndBeforeDeathDate'
+  /**
+   * @deprecated This validator is deprecated and will be removed in future releases. Use `isAgeInYearsBetween` instead
+   */
   | 'isInformantOfLegalAge'
```

Comment:
@Siyasanga its fine for the line to be deleted entirely. The CHANGELOG message is enough for countries to update

Reply:
@Siyasanga you might aswell remove the deprecated comment and the "isInformantOfLegalAge" type too.